### PR TITLE
Fix kafka consumer connection leak

### PR DESCRIPTION
**Pull request by jaime0815**
issue #27222

### CODE REVIEW

**Comment by jaime0815** [comment]
pr: https://github.com/milvus-io/milvus/pull/27224

File: Makefile [hunk 1]
```diff
@@ -174,7 +174,7 @@ generated-proto: download-milvus-proto build-3rdparty
 	@(env bash $(PWD)/scripts/generate_proto.sh)
 
 build-cpp: generated-proto
-	@echo "Building Milvus cpp library ... ${AZURE_OPTION}"
```
Comment (czs007) [inline Makefile:177]: what's this?

**Pull request merged**